CONTRIBUTING.rst

```diff
@@ -25,42 +23,118 @@ If you are proposing a feature:
 
 
 Get Started!
-------------
+============
 
-Ready to contribute? Here's how to set up `earthpy` for local development.
+Ready to contribute? Here's how to set up EarthPy for local development.
 
-1. Fork the `earthpy` repo on GitHub.
-2. Clone your fork locally::
+1. Fork the repository on GitHub
+--------------------------------
 
-    $ git clone git@github.com:your_name_here/earthpy.git
+To create your own copy of the repository on GitHub, navigate to the
+`earthlab/earthpy <https://github.com/earthlab/earthpy>`_ repository
+and click the **Fork** button in the top-right corner of the page.
+
+2. Clone your fork locally
+--------------------------
 
-3. Set up your fork for local development with conda::
+Use ``git clone`` to get a local copy of your EarthPy repository on your
+local filesystem::
 
+    $ git clone git@github.com:your_name_here/earthpy.git
```

Comment:
hey @mbjoseph do most people use ssh? i don't ! maybe i should

Reply:
Not sure - we can always add instructions to clone both ways later on!